lib/staccato/v4/events/share.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
It's wild to me that ruby still requires this 😆

Reply:
Yeah, I really wish it had become the default in ruby 3.0!  Maybe in 3.3?

Reply:
🤞🏼